src/libraries/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -39,8 +39,8 @@
     <Compile Include="System\Net\Sockets\SocketOptionLevel.cs" />
     <Compile Include="System\Net\Sockets\SocketOptionName.cs" />
     <Compile Include="System\Net\Sockets\SocketShutdown.cs" />
-    <Compile Include="System\Net\Sockets\TCPClient.cs" />
-    <Compile Include="System\Net\Sockets\TCPListener.cs" />
+    <Compile Include="System\Net\Sockets\TcpClient.cs" />
+    <Compile Include="System\Net\Sockets\TcpListener.cs" />
```

Comment:
This is failing the Unix builds, since the files were not renamed. Let's avoid changes unrelated to the core functionality in the PR.